network-api/networkapi/wagtailpages/migrations/0030_populate_profile_slug.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Kevin on 2022-06-17 09:55
+
+import django.db.models.deletion
+import modelcluster.fields
+from django.db import migrations, models
+from networkapi.wagtailpages.models import Profile
```

Comment:
**Question** Isn't this one of those cases where we need to use `apps.get_model` rather than importing directly? Or would this work because we are not setting the field directly by are relying on the save method to do the migration for us? 

https://docs.djangoproject.com/en/4.0/topics/migrations/#data-migrations

Reply:
Have altered this in 1331e9dd04eb6152f3eb60f9dfb1cc51d77fe48c

Good spot - it will work, be we probably don't want to (can't) rely on the model save method here so I've updated it to mimic what we expect on save